packages/docs-app/src/examples/core-examples/inputGroupExample.tsx

```diff
@@ -37,16 +38,18 @@ export class InputGroupExample extends React.PureComponent<IExampleProps, IInput
         filterValue: "",
         large: false,
         showPassword: false,
-        tagValue: "",
+        small: false,
+        tagValue: ""
```

Comment:
pretty sure our linter wants trailing commas. you are not respecting the local config.

Reply:
my bad. didn't see the lint warnings. fixed thoser